<launch>

    <arg name="output" default="screen"/>
    <arg name="tracker_hostname" default="10.10.10.5"/>
    <arg name="tracker_port" default="801"/>
    <arg name="tracker_name" default="mocap"/>
    <arg name="object_tracker_topic" default="mocap_tracking"/>
    <arg name="marker_tracker_topic" default="mocap_marker_tracking"/>
    <arg name="tracker_origin_frame_name" default="mocap_world"/>
    <arg name="stream_mode" default="ServerPush"/>
    <arg name="override_timestamps" default="false"/>
    <arg name="use_sim_time" default="false"/>

    <arg name="publish_raw_markers" default="false"/>
    <arg name="publish_label_markers" default="false"/>
    <!-- <arg name="tracking_object_name" default="Page1"/> -->

    <node pkg="lightweight_vicon_bridge" exec="lightweight_vicon_bridge_node" name="lightweight_vicon_bridge" output="$(var output)">
<!--         <param name="tracker_hostname" value="$(var tracker_hostname)"/> -->
<!--         <param name="tracker_port" value="$(var tracker_port)"/> -->
<!--         <param name="tracker_frame_name" value="$(var tracker_origin_frame_name)"/> -->
<!--         <param name="tracker_name"  value="$(var tracker_name)"/> -->
<!--         <param name="tracker_topic"  value="$(var object_tracker_topic)"/> -->
<!--         <param name="stream_mode" value="$(var stream_mode)"/> -->
<!--         <param name="use_sim_time" value="$(var use_sim_time)"/> -->
    </node>

    <node if="$(var publish_raw_markers)" pkg="lightweight_vicon_bridge" exec="raw_marker_bridge_node" name="raw_marker_bridge" output="$(var output)">
<!--         <param name="tracker_hostname"  value="$(var tracker_hostname)"/> -->
<!--         <param name="tracker_port"  value="$(var tracker_port)"/> -->
<!--         <param name="tracker_frame_name"  value="$(var tracker_origin_frame_name)"/> -->
<!--         <param name="tracker_name" value="$(var tracker_name)"/> -->
<!--         <param name="tracker_topic" value="$(var marker_tracker_topic)"/> -->
<!--         <param name="stream_mode"  value="$(var stream_mode)"/> -->
<!--         <param name="use_sim_time"  value="$(var use_sim_time)"/> -->
    </node>

<!--     <node if="$(arg publish_label_markers)" pkg="lightweight_vicon_bridge" type="raw_label_marker_bridge_node" name="raw_label_marker_bridge" output="$(arg output)"> -->
<!--         <param name="tracker_hostname" type="string" value="$(arg tracker_hostname)"/> -->
<!--         <param name="tracker_port" type="string" value="$(arg tracker_port)"/> -->
<!--         <param name="tracker_frame_name" type="string" value="$(arg tracker_origin_frame_name)"/> -->
<!--         <param name="tracker_name" type="string" value="$(arg tracker_name)"/> -->
<!--         <param name="tracker_topic" type="string" value="$(arg marker_tracker_topic)"/> -->
<!--         <param name="stream_mode" type="string" value="$(arg stream_mode)"/> -->
<!--         <param name="use_sim_time" type="bool" value="$(arg use_sim_time)"/> -->
<!--          --><!-- <param name="tracking_object_name" type="string" value="$(arg tracking_object_name)"/> -->
<!--     </node> -->

<!--     <node pkg="lightweight_vicon_bridge" type="tf_publisher_node" name="mocap_tf_publisher_node" output="$(arg output)"> -->
<!--         <param name="tracker_topic" type="string" value="$(arg object_tracker_topic)"/> -->
<!--         <param name="override_timestamps" type="bool" value="$(arg override_timestamps)"/> -->
<!--         <param name="use_sim_time" type="bool" value="$(arg use_sim_time)"/> -->
<!--     </node> -->

<!--       <node pkg="tf2_ros" type="static_transform_publisher" name="victor_torso_mocap_tf_shim_broadcaster" required="true" -->
<!--               args=" -->
<!--                 -0.017089397889861514 0.03745940545591349 -1.4090475155142663 -->
<!--                 0.015541007504028821 0.0069256832595382635 0.020888248743866845 0.9996370306574095 -->
<!--                     mocap_VictorTorso1_VictorTorso1 world"> -->
<!--             <param name="use_sim_time" value="$(arg use_sim_time)"/> -->
<!--         </node> -->

</launch>




